docs/resources/Channel.md

```diff
@@ -25,7 +25,8 @@ Represents a guild or DM channel within Discord.
 | owner_id?              | snowflake                                                              | id of the DM creator                                                                                                                                                            |
 | application_id?        | snowflake                                                              | application id of the group DM creator if it is bot-created                                                                                                                     |
 | parent_id?             | ?snowflake                                                             | id of the parent category for a channel (each parent category can contain up to 50 channels)                                                                                    |
-| last_pin_timestamp?    | ?ISO8601 timestamp                                                     | when the last pinned message was pinned. This may be `null` in events such as `GUILD_CREATE` when a message is not pinned.                                                  |
+| last_pin_timestamp?    | ?ISO8601 timestamp                                                     | when the last pinned message was pinned. This may be `null` in events such as `GUILD_CREATE` when a message is not pinned.                                                      |
+| video_quality_mode?    | integer                                                                | the camera [video quality mode](#DOCS_RESOURCES_CHANNEL/channel-object-video-quality-modes) of the voice or stage channel, 1 when not present                                   |
```

Comment:
stage channels dont have video.

Reply:
they still have the property and the setting in the client

![image](https://user-images.githubusercontent.com/11778454/113219459-4a74f400-9236-11eb-87de-9399e81c27b9.png)

```js
{
  id: '826946920634253362',
  type: 13,
  name: 'stage-mode',
  position: 1,
  parent_id: null,
  topic: null,
  bitrate: 40000,
  user_limit: 1000,
  rtc_region: null,
  video_quality_mode: 2,
  guild_id: '252296452708106240',
  permission_overwrites: [],
  nsfw: false
}
```

Reply:
That's a client bug. Let's not propagate that to the docs too.